src/App.tsx

```diff
@@ -26,70 +15,141 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+  let visibleGoods = [...goods];
+
+  if (sortType === 0 && !isReversed) {
```

Comment:
```suggestion
  switch (sortType) {
    case SortType.NONE
```
use enum type and switch/case instead of ifs
condition && !isReversed is useless here, you have reverse check below